### PR TITLE
added Hyundai Santafe limited 2020 FP in hyundai values and updated description for SANTA_FE

### PR DESCRIPTION
**Pull request by newstackdevelopment**

### CODE REVIEW

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ class CAR:
   IONIQ_EV_2020 = "HYUNDAI IONIQ ELECTRIC 2020"
   KONA = "HYUNDAI KONA 2020"
   KONA_EV = "HYUNDAI KONA ELECTRIC 2019"
-  SANTA_FE = "HYUNDAI SANTA FE LIMITED 2019"
+  SANTA_FE = "HYUNDAI SANTA FE LIMITED 2019-2020"
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:31]: revert this, no need to update it

Comment (newstackdevelopment) [reply selfdrive/car/hyundai/values.py]: has been reverted back to only 2019

**Review by newstackdevelopment — COMMENTED** [review]
reverted the description change back to only 2019

**Comment by adeebshihadeh** [comment]
What's your Dongle ID?

**Comment by VirtuallyChris** [comment]
Which harness type does this car use? Still Hyundai D?

**Comment by newstackdevelopment** [comment]
Yes still D but I pushed it to my comma and am getting a unable to import error

**Comment by newstackdevelopment** [comment]
My error was not related to the pull request. I pushed this branch into my device and this branch is missint a steeringlimits class and it was replace with carparams class. So my pull request is still good to go

File: selfdrive/car/hyundai/values.py (hunk 1 above)
Comment (newstackdevelopment) [reply selfdrive/car/hyundai/values.py]: Any news on this. It has fixed my issue locally. Am I missing anything else to get this approved?

**Pull request merged**